cumulusci.yml

```diff
@@ -118,7 +117,21 @@ tasks:
             values: "Administrative Offices Area,Athletic Field or Playground,Auditorium,Bus Stop,Cafeteria Area,Classroom,Computer Lab,Hallway or Stairs,Library/Media Center,Locker Room or Gym Areas,Off Campus,On Campus,Online,Parking Lot,Restroom,School Bus,Stadium,Unknown,Walking to or from School"
             existing_values: "Off Campus, On Campus"
             recordtypes: Incident
-            restricted: true
+            restricted: True
+            sorted: True
+            namespaced: False
+    
+    add_case_category_values:
+        description: Adds additional picklist values to the Category field.
+        class_path: tasks.add_picklist_values.AddPicklistValues
+        group: 'K-12: Metadata'
+        options:
+            sobject: Case
+            field: hed__Category__c
+            values: "Cooperation,Integrity,Leadership,Respect,School Code of Conduct,School Violation,State Offense,Teamwork"
```

Comment:
@kdy618 The result for this is that Positive Behavior for case has all of the values listed instead of the values specified in the spec. Please change this so we may have only the desired record types for Positive Behavior cases.

Reply:
@MatthewBlanski , although I have removed the 3 extra values, I'm still seeing it on the selected value column on the record type. I've communicated this to @jofsky  as well on slack.